pyproject.toml

```diff
@@ -42,6 +42,8 @@ dependencies = [
   "napari-psf-analysis",
   "imageio-ffmpeg",
   "dask[array]",
+  "toml",
+  "ultrack @ git+https://github.com/royerlab/ultrack.git"
```

Comment:
maybe add a git hash here to make sure you're always using the same ultrack version?

Reply:
What is needed from the main branch that is not released?